changelog/unreleased/issue-148

```diff
@@ -0,0 +1,7 @@
+Bugfix: Enhance security for example systemd unit file
+
+Additional systemd features have been used to enhance the security of the
```

Comment:
That sounds a bit like the rest-server was insecure before. But these changes are rather a hardening against potentially existing bugs.